Makefile

```diff
@@ -4,7 +4,7 @@ KIND ?= kind
 DOCKER ?= docker
 DOCKER_ARGS ?= --load
 APP_NAME ?= ghcr.io/nccloud/metadata-reflector
-TAG ?= 0.1.0-dev
+TAG ?= 0.1.1-dev
```

Comment:
could you please update it to `0.2.1-dev`? My bad, I didn't update it in the latest PR :disappointed: but this is the expected release.